mailpoet/lib/Automation/Integrations/MailPoet/Payloads/OrderPayload.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Integrations\MailPoet\Payloads;
+
+use MailPoet\Automation\Engine\Integration\Payload;
+
+class OrderPayload implements Payload {
+
+
+  private $order;
+
+  public function __construct(
+    \WC_Order $order
+  ) {
+    $this->order = $order;
+  }
+
+  public function getOrder(): \WC_Order {
+    return $this->order;
+  }
+
+  public function getEmail(): string {
+    return $this->order->get_billing_email();
+  }
+
+  public function getId(): int {
+    return $this->order->get_id();
+  }
+
+  public function getFormattedOrderStatus(): string {
+    return 'wc-' . $this->order->get_status();
```

Comment:
Why is the `wc-` prefix needed? Why not using the Woo-original naming?

Reply:
Woo itself is inconsistent. Either we add the `wc-` prefix here or we remove it from the `wc_get_order_statuses()` result.